components/ItemGrid/LoadItemsTask2.bs

```diff
@@ -264,7 +264,7 @@ sub loadItems()
                     seasonName = ` - ${item.LookupCI("SeasonName")}`
 
                     if isValid(item.LookupCI("IndexNumber"))
-                        seasonName += ` Episode ${item.LookupCI("IndexNumber")}`
+                        seasonName += ` tr("Episode") ${item.LookupCI("IndexNumber")}`
```

Comment:
```suggestion
                        seasonName += ` ${tr("Episode")} ${item.LookupCI("IndexNumber")}`
```

This line is incorrectly using template literals and is not getting translated.
![image](https://github.com/user-attachments/assets/c80199b9-051d-43a8-933c-46fd82677122)